### DISCUSSION TITLE
Form.List add multiple fields at once

### DISCUSSION THREAD

**Discussion by nichtsam** [ant-design/ant-design#36236] [Q&A]
The Form.List operation's `add` currently only supports to add one field at a time, while `remove` supports to remove an array of indices.
I search a bit but saw nothing about this topic.
Is there a way to add multiple fields at once in Form.List?

Now the only way I figured is to make a loop, but it will cost lots of validations and thus performance issue.
Thanks.

**Comment by nichtsam** [comment]
In case anyone needs an alternative,
I use set `form.setFields` to directly set it.

**Reply by vkot91** [reply]
Can you please provide an example?

**Reply by nichtsam** [reply]
> Can you please provide an example?

Hi, sorry, it's been long time ago and I'm not using antd anymore.
I took a look at the doc, I think you could just make an array of the fields you want, and pass it to `form.setFields`.
